interop-cats/jvm/src/test/scala/zio/interop/Fs2ZioSpec.scala

```diff
@@ -22,7 +18,7 @@ object Fs2ZioSpec extends DefaultRunnableSpec {
           }
         },
         testM("works if F is zio.interop.Task") {
-          testCaseJoin[zio.Task].map { ints =>
+          testCaseJoin[zio.RIO[ZEnv, *]].map { ints =>
```

Comment:
Can we provide an instance for `Task` that uses the live services?

Reply:
I can give it a try but I suspect implicit priorities could be messed up due to contravariance

Reply:
I added instances that depend on the runtime: 28b4d37

Reply:
Nice! Any implicit prioritization issues with defining those instances at the same level as the more polymorphic ones?

Reply:
Not really. Note that those instances fix the environment to `Any`. Alternatively we could try to leave it generic, but I'm not convinced - usually you have either no environment or you have at least `ZEnv` and add more on top. WDYT?

Reply:
I think it is fine.